code/game/objects/items/mail.dm

```diff
@@ -332,3 +334,166 @@
 /obj/item/paper/fluff/junkmail_generic/Initialize(mapload)
 	default_raw_text = pick(GLOB.junkmail_messages)
 	return ..()
+
+// =====================
+// Antag stuff down here
+// =====================
+
+/obj/item/mail/traitor
+	var/armed = FALSE
+	var/mob/madeby
+	goodie_count = 0
+
+/obj/item/mail/traitor/envelope
+	name = "envelope"
+	icon_state = "mail_large"
+	stamp_max = 2
+	stamp_offset_y = 5
+
+/obj/item/mail/traitor/attack_self(mob/user)
+	if(recipient_ref)
```

Comment:
When is this possible to be false? I thought these were part of normal init?

Reply:
i dont really know. Not sure, should i change it.

Reply:
If there's no case where this isn't set before attack_self, then we shouldn't check for it